pkg/cmd/builder/build.go

```diff
@@ -42,6 +42,21 @@ import (
 	"github.com/containerd/platforms"
 )
 
+type PlatformParser interface {
```

Comment:
why do we need to introduce a new wrapper form platforms pkg?

Reply:
wanted to add a unit test via mocking the platform package, seemed like the preferred way to do a dependency injection to facilitate mocking.